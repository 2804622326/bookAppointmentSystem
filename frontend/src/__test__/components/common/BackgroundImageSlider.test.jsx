import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import BackgroundImageSlider from '../../../components/common/BackgroundImageSlider.jsx';

test('renders background slider and matches snapshot', () => {
  const { asFragment } = render(<BackgroundImageSlider />);
  expect(asFragment()).toMatchSnapshot();
});
