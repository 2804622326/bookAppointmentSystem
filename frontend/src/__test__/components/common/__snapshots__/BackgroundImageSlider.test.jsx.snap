// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders background slider and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="background-slider"
  >
    <div
      class="carousel slide"
    >
      <div
        class="carousel-indicators"
      >
        <button
          aria-current="true"
          aria-label="Slide 1"
          class="active"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 2"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 3"
          data-bs-target=""
          type="button"
        />
      </div>
      <div
        class="carousel-inner"
      >
        <div
          class="active carousel-item"
        >
          <img
            alt="Slide 0"
            class="d-block w-100"
            src="test-file-stub"
          />
        </div>
        <div
          class="carousel-item"
        >
          <img
            alt="Slide 1"
            class="d-block w-100"
            src="test-file-stub"
          />
        </div>
        <div
          class="carousel-item"
        >
          <img
            alt="Slide 2"
            class="d-block w-100"
            src="test-file-stub"
          />
        </div>
      </div>
      <a
        class="carousel-control-prev"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
        <span
          class="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        class="carousel-control-next"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
        <span
          class="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
